db/ExpectionService/index.js

```diff
@@ -0,0 +1,13 @@
+const BaseService = require("../BaseService");
+const ExpectionModel = require("./ExpectionModel");
+
+class ExpectionService extends BaseService {
+    constructor() {
+        super();
+
+        this.name = "expectation";
```

Comment:
instead of setting this specifically yourself on this, I think this should be provided to the costructor